workspaces/Cargo.toml

```diff
@@ -32,6 +32,7 @@ tracing = "0.1"
 url = { version = "2.2.2", features = ["serde"] }
 
 near-gas = { version = "0.2.3", features = ["serde", "borsh", "schemars"] }
+near-token = { version = "0.1.0", features = ["serde", "borsh", "schemars"] }
```

Comment:
Do we really need schemars and borsh features?

On a separate note, remove near-units from dev-dependecies